README.md

```diff
@@ -1,36 +1,99 @@
+
+```bash
+npx expo prebuild
+```
+
+## Install in bare React Native project
+
+For bare React Native projects, [install and configure the `expo` package](https://docs.expo.dev/bare/installing-expo-modules/) before continuing.
+
+### Add the package to your npm dependencies
+
+<!--eng: Should this command use xmtp-react-native or xmtp-react-native-sdk? When I run this command, I get npm ERR! 404 Not Found - GET https://registry.npmjs.org/xmtp-react-native - Not found. This is because we haven't submitted to npm yet, correct? Will this command work for pre-preview?-->
+
+```bash
 npm install xmtp-react-native-sdk
 ```
 
 ### Configure for iOS
 
-Run `npx pod-install` after installing the npm package.
+After installing the npm package, run:
+
+```bash
+npx pod-install
+```
 
 ### Configure for Android
 
-# Contributing
+<!--eng: Seems weird to have a section to say that there's nothing to do, but thought it might be helpful to explictly call this out. If we leave it out, folx might think the Android info is missing. WDYT?-->
+
+No configuration necessary.
+
+## XMTP `production` and `dev` network environments
+
+XMTP provides both `production` and `dev` network environments to support the development phases of your project.
+
+The `production` and `dev` networks are completely separate and not interchangeable.
+
+For example, for a given blockchain account address, its XMTP identity on `dev` network is completely distinct from its XMTP identity on the `production` network, as are the messages associated with these identities. In addition, XMTP identities and messages created on the `dev` network can't be accessed from or moved to the `production` network, and vice versa.
+
+**Important:** When you create a client, it connects to the XMTP `dev` environment by default. You can use the `env` parameter to set your client's network environment.
+
+The `env` parameter accepts one of three valid values: `dev`, `production`, or `local`. Here are some best practices for when to use each environment:
+
+- `dev`: Use to have a client communicate with the `dev` network. As a best practice, set `env` to `dev` while developing and testing your app. Follow this best practice to isolate test messages to `dev` inboxes.
+
+- `production`: Use to have a client communicate with the `production` network. As a best practice, set `env` to `production` when your app is serving real users. Follow this best practice to isolate messages between real-world users to `production` inboxes.
+
+- `local`: Use to have a client communicate with an XMTP node you are running locally. For example, an XMTP node developer can set `env` to `local` to generate client traffic to test a node running locally.
```

Comment:
FWIW, none of this currently works in this project (tracked in https://github.com/xmtp/xmtp-react-native/issues/4).